src/vm/threads.cpp

```diff
@@ -303,8 +303,8 @@ bool Thread::DetectHandleILStubsForDebugger()
 extern "C" {
 #ifndef __llvm__
 __declspec(thread)
-#else // !__llvm__
-__thread 
+#else // __llvm__
```

Comment:
The comment should have the `!` in it. The convention is to put the same comment on `#if`, `#else` and `#end`